### PR TITLE
add contact and footer sections

### LINKED ISSUE #11: Add Contact and Footer sections
Sections should look something like the picture below but are consistent with the existing content of the website.
contact and footer sections

### PR DESCRIPTION
**Pull request by hugh-logan**
Closes #11

### CODE REVIEW

**Comment by hugh-logan** [comment]
Please review @jagannathBhat

**Comment by jagannathBhat** [comment]
Please make these changes as well.

- [ ] Remove discord link from the `Footer`. It's already shown in the `Contact` section.
- [ ] Add `max-width` styling to the footer as well. Same value as the `Blog` section.

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { SOCIALS } from '../About/constants'
```
Comment (jagannathBhat) [inline src/components/Footer/index.js]: Please create a `constants.js` file in the components folder and move the `SOCIALS` constant to that file. Please make sure you update other files that import `SOCIALS`.

**Review by jagannathBhat — CHANGES_REQUESTED** [review]

**Comment by hugh-logan** [comment]
Please review @jagannathBhat

**Review by jagannathBhat — APPROVED** [review]

**Pull request merged**